admin/app/model/AdminLifecycle.scala

```diff
@@ -118,10 +110,7 @@ class AdminLifecycle(appLifecycle: ApplicationLifecycle,
     jobs.deschedule("FastlyCloudwatchLoadJob")
     jobs.deschedule("R2PagePressJob")
     jobs.deschedule("AnalyticsSanityCheckJob")
-    jobs.deschedule("FrontPressJob")
     jobs.deschedule("RebuildIndexJob")
-    jobs.deschedule("MatchDayRecorderJob")
-    jobs.deschedule("SentryReportJob")
```

Comment:
yes. Not related to this change but they have been removed long time ago.

https://github.com/guardian/frontend/commit/4d56f1a839af6c28587dad2ec49b45c56f1e3494#diff-cd3dba3d84959613e581688dbb7558f1L63
https://github.com/guardian/frontend/commit/39aac6d4528802bdb92f82f7ee21e82eee37a092